src/main/java/commands/ListModuleCommand.java

```diff
@@ -0,0 +1,47 @@
+package commands;
+
+import access.Access;
+import manager.admin.ModuleList;
+import manager.module.Module;
+import storage.Storage;
+import ui.Ui;
+
+import java.util.ArrayList;
+
+import static common.Messages.MODULE;
+
+public class ListModuleCommand extends ListCommand {
+    public static final String COMMAND_WORD = "list";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Shows a list of modules available. \n"
+            + "Example: " + COMMAND_WORD + "\n";
```

Comment:
Perhaps can change chapters to %ss in ListCommand and when you use this message, Stringformat with the type Messages.CHAPTER